int32 mod_freq_value
---
int32 mod_freq_new_value
